db/memtable.cc

```diff
@@ -375,6 +375,7 @@ class MemTableIterator : public InternalIterator {
     PERF_COUNTER_ADD(next_on_memtable_count, 1);
     assert(Valid());
     iter_->Next();
+    TEST_SYNC_POINT_CALLBACK("MemTableIterator::Next:0", iter_);
```

Comment:
Just curious as to what this has to do with this change?

Reply:
I didn't find a good way to fake a memtable that generates out of order keys. If there is a better way, will do it.